src/pageObjects/garagePage/GaragePage.js

```diff
@@ -0,0 +1,10 @@
+import BasePage from "../BasePage.js";
+
+export default class GaragePage extends BasePage {
+
+    constructor(page) {
+        super(page, '/panel/garage', page.locator('app-panel-layout' , {has : page.locator('button', {hasText: 'Add car'})}));
+        this.activeHeaderItem = page.locator('a.btn.header-link.-active')
```

Comment:
It should be either in Header component or in BasePage. First option is preferable